source/documentation/guidance/buildpacks.md

```diff
@@ -31,18 +31,17 @@ Regularly maintain your buildpack to update your runtime and dependencies as new
 
 #### Docker Images
```

Comment:
This should be ```Docker images``` (I -> i)

Reply:
Yeah, I've mentioned this to @jonathanglassman already; he's going to fix up these things separately.